packages/vite/src/node/plugins/worker.ts

```diff
@@ -49,10 +49,32 @@ function saveEmitWorkerAsset(
   workerMap.assets.set(fileName, asset)
 }
 
+// Ensure that only one rollup build is called at the same time to avoid
+// leaking state in plugins between worker builds.
```

Comment:
```suggestion
// Ensure that only one rollup build is called at the same time to avoid
// leaking state in plugins between worker builds. 
// TODO: Review if we can parallelize the bundling of workers.
```